index.html

```diff
@@ -97,6 +97,7 @@ <h2 id="mainnet">mainnet</h2>
 <li><a href="https://raw.githubusercontent.com/element-fi/elf-deploy/main/changelog/releases/mainnet/v1.0.2:17/addresses.json">v1.0.2:17</a></li>
 <li><a href="https://raw.githubusercontent.com/element-fi/elf-deploy/main/changelog/releases/mainnet/v1.0.2:18/addresses.json">v1.0.2:18</a></li>
 <li><a href="https://raw.githubusercontent.com/element-fi/elf-deploy/main/changelog/releases/mainnet/v1.0.2:19/addresses.json">v1.0.2:19</a></li>
+<li><a href="https://raw.githubusercontent.com/element-fi/elf-deploy/main/changelog/releases/mainnet/v1.0.3:1/addresses.json">v1.0.3:1</a></li>
```

Comment:
Should this be v1.1.0:1?

Reply:
Correct! Updated it to the correct version